### DISCUSSION TITLE
Provide a simple way to close tabs when vertical tabs just show icons

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#834] [Ideas]
### What feature would you like to see?

Currently, when the sidebar is configured to only show icons, there is no way to close tabs without right clicking and selecting close from the list of options.

A more elegant solution might be to have an 'x' pop up on the top right of each tab icon.

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/443_

**Comment by Ophitoxaemia** [comment]
Add my vote for a quick way to close tabs. I tend to open and close a lot of tabs. I'm on Linux too.
